src/keypair.rs

```diff
@@ -241,20 +231,20 @@ impl Keypair {
         &self,
         prehashed_message: D,
         context: Option<&[u8]>,
-    ) -> Signature
+    ) -> ed25519::Signature
```

Comment:
This is now returning the `Signature` type from the `ed25519` crate (which is re-exported as `ed25519_dalek::Signature`).

I wasn't sure if this was helpful or not to signal this is the `ed25519` crate's signal type versus an `InternalSignature`.